maven_iuvs/instrument.py

```diff
@@ -47,6 +47,9 @@
 slit. This is out of 1024 pixels (index 1023) for a 1024x1024 pixel
 detector."""
 
+ech_slit_start = 346  # Starting pixel of slit in echelle mode
```

Comment:
slit position for H/D Lyman alpha

Reply:
Oh yeah, thanks. You would think I would know this having looked at the images 293752 times.

Reply:
Fixed by 0170b4e961f6c74d1ac2a1ea7d407fde769d4baf